Felienne/hedy
# [BUG FIX] Fixes valid token for password reset

**Description**
In this PR we fix the reset of a password using the reset function. Currently the token creation to valid the reset does not work, as the values are stored in the database incorrectly. This PR should fix that.

**Fixes**
No related issue.

**How to test**
Deploy to Alpha and verify that password reset is working and not the "token invalid" error is returned.

**Checklist**
Done? Check if you have it all in place using this list*
  

* If you're unsure about any of these, don't hesitate to ask. We're here to help!
